app/views/runs/stats/index.slim

```diff
@@ -112,12 +112,20 @@ article#stats
                 span.time -
           tr
             td
-              span Total Playtime
+              span Total playtime
             td
               - if @run.total_playtime_ms(timing) != 0
                 span = format_ms(@run.total_playtime_ms(timing))
               - else
                 span.time -
+          tr
+            td
+              span Total pause time
```

Comment:
"Total playtime" means sum of all time put into the game across all runs, so having "total pause time" right next to it might make people think it's the sum of all time put into pauses across all runs.

Reply:
Oh, this was total pause time across all runs.  I sum all of them from all the run attempts.  Did you want just for this run, the sum, or both?

Reply:
Ohh, didn't notice that, my bad. What's the use case for seeing total pausetime?

Reply:
Uh, my lack of reading comprehension when looking at the issue tracker...